### PR TITLE
feat(improve-nav-bar): improves nav-bar

### PR DESCRIPTION
**Pull request by biantris**
image

### CODE REVIEW

File: src/data/data.tsx [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     path: "/destination",
   },
   {
-    name: "02 Crew",
+    name: "02 CREW",
     path: "/crew",
   },
   {
-    name:  "03 Technology",
+    name:  "03 TECHNOLOGY",
```
Comment (eryc-cc) [inline src/data/data.tsx:12]: Why not leave the text capitalized and change its look with `text-transform`?

Comment (biantris) [reply src/data/data.tsx:12]: it's a good way too, i just avoided adding more stylization

Comment (eryc-cc) [reply src/data/data.tsx:12]: Any specific reason why you wouldn’t want to add more stylization? The reason I ask is bc I want to know the thought process that goes into building/styling with CSS-in-JS or `.tsx` files 😁 It’s ok if it’s just like “ah, it’s a personal project, I didn’t wanna lose time” or something

Comment (biantris) [reply src/data/data.tsx:12]: As I am using a js framework (next.js) and as the proposed challenge is to create the components "by hand" I found it easier for me to use styled-components, but pure CSS would be nice if the project was "simpler".
I believe that the choice is something more personal and depends on the project, I sent this question here and some friends answered: https://twitter.com/biantris_/status/1518559767031070722?s=20&t=dpfIrQusNUUJhcjKZ02zHw

**Pull request merged**